### PR TITLE
[Example] Object Detection UI upgrade

### LINKED ISSUE #546: [Example] Code refactor for Object Detection

### PR DESCRIPTION
**Pull request by ibelem**
Fix #546 , Test URL: https://ibelem.github.io/webml-website/examples/object_detection/index.html 

@Wenzhao-Xiang @pinzhenx @huningxin PTAL.

### CODE REVIEW

**Comment by ibelem** [comment]
@Wenzhao-Xiang is still working on Object Detection with more models support in #480 , it's OK to merge this PR after #480.

Known issue in utils.js: There is no model loaded status check in this utils.js, unexpected loading progress and results will be shown when switching to other models during model loading. Will add fix in utils.js with #480.

**Comment by Wenzhao-Xiang** [comment]
I have just updated #480 to the newest version. Now I'm waiting for API updating, then I can support the new op in polyfill.

**Comment by ibelem** [comment]
@Wenzhao-Xiang, @huningxin will attend the Tensorflow Dev Summit next week so we need to publish new UI online at first, please review the UI and code for the Object Detection without Yolo relevant models updates.   
https://ibelem.github.io/webml-website/examples/object_detection/index.html

File: examples/util/base.js [hunk 1]
```diff
@@ -236,12 +240,13 @@ const ssd_mobilenetv2_tflite = {
...
     mean: [127.5, 127.5, 127.5],
     std: [127.5, 127.5, 127.5],
-  }
-};
-
-const ssdlite_mobilenetv2_tflite = {
-  modelName: 'SSDLite MobileNet v2(TFlite)',
-  modelSize: '67.3MB',
+  },
+  intro: 'MobileNet V2 with SSD. This model is faster than Mobilenet V1 SSD at the same accuracy.',
```
Comment (Wenzhao-Xiang) [inline examples/util/base.js]: image
image
I think SSD Mobilenet V2 is slower than V1, but have a higher accuracy.

File: examples/util/base.js [hunk 2]
```diff
@@ -251,11 +256,14 @@ const ssdlite_mobilenetv2_tflite = {
   preOptions: {
     mean: [127.5, 127.5, 127.5],
     std: [127.5, 127.5, 127.5],
-  }
-};
+  },
+  intro: 'MobileNet V2 with SSDLite. This model is faster than Mobilenet V1 SSD at the same accuracy.',
```
Comment (Wenzhao-Xiang) [inline examples/util/base.js]: image
image
Compared with SSD Mobilenet V2, SSDLite Mobilenet V2 is much faster, and almost don't lose the accuracy.

Comment (Wenzhao-Xiang) [reply examples/util/base.js]: intro: 'MobileNet V2 with SSDLite, which is an upgraded version of SSD MobileNet V2. Compared with SSD Mobilenet V2, SSDLite Mobilenet V2 is much faster, and almost have no loss of the accuracy. '

File: examples/util/base.js (hunk 1 above)
Comment (Wenzhao-Xiang) [reply examples/util/base.js]: intro: 'MobileNet V2 with SSD. This model is slower than Mobilenet V1 SSD, but have a higher accuracy. '

**Comment by ibelem** [comment]
Merge this PR.

**Pull request merged**